wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -17,19 +17,25 @@
 package com.android.identity_credential.wallet
 
```

Comment:
Looks like none of the changes in this file are necessary? Unused imports and whitespace changes....

Reply:
Yep.